### PR TITLE
[OPS-7732] migrate from DockerHub to GitHub Actions && AWS ECR

### PR DESCRIPTION
**Pull request by teodorescuserban**

### CODE REVIEW

**Review by alexandru-m-g — COMMENTED** [review]
Could we name the github workflow files similarly ? 
- either all yml or all yaml
- create-py3-image.yaml & docker-build-image.yml are doing the same thing one for PY3 another for PY2. Could we name them similarly ?

File: .github/workflows/docker-build-image.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Login to Public ECR
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```
Comment (alexandru-m-g) [inline .github/workflows/docker-build-image.yml]: Can we use a specific tag/versino of the _docker/login-action_ command ? If not, could you please add a comment with why we need this specific version ?

Comment (teodorescuserban) [reply .github/workflows/docker-build-image.yml]: yep sure, pushed to use the generic v1

**Comment by teodorescuserban** [comment]
> Could we name the github workflow files similarly ?
> 
> * either all yml or all yaml
> * create-py3-image.yaml & docker-build-image.yml are doing the same thing one for PY3 another for PY2. Could we name them similarly ?

- all yml is fine. fixed.
- yeah, i know, the "create-py3-image" would phase out once we upgrade to python3. in the mean time, the build yml will only trigger on tags, dev and prod.

cc @alexandru-m-g

**Comment by alexandru-m-g** [comment]
Closing this PR in favor of #5585 (has same code)

**Pull request closed without merging**